From 1f9c1f8acbcafc9bf5ab5d20399609c3886e1cc2 Mon Sep 17 00:00:00 2001
From: Suryashankar Das <suryashankardas.2002@gmail.com>
Date: Tue, 16 Mar 2021 18:21:05 +0530
Subject: [PATCH] overview: Pass state while calling _animateVisible()

Pass OverviewControls.ControlsState.WINDOW_PICKER to the _animateVisible
function in _hideDone().

Fixes: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3876
---
 js/ui/overview.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/overview.js b/js/ui/overview.js
index 75fe6e1a8f..ca5de0c226 100644
--- a/js/ui/overview.js
+++ b/js/ui/overview.js
@@ -619,7 +619,7 @@ var Overview = class {
         this.emit('hidden');
         // Handle any calls to show* while we were hiding
         if (this._shown)
-            this._animateVisible();
+            this._animateVisible(OverviewControls.ControlsState.WINDOW_PICKER);
         else
             Main.layoutManager.hideOverview();
 
-- 
GitLab

